app/src/shared/utils/time.js

```diff
@@ -5,31 +5,31 @@ const units = [
         max: 2760000,
         value: 60000,
         name: 'minute',
-        prev: 'a minute ago',
+        prev: 'A minute ago',
```

Comment:
This doesn't work nicely when we combine usage with a verb like "Updated A minute ago".

Reply:
Ah, good catch. Will fix.

Reply:
Addressed in https://github.com/streamr-dev/streamr-platform/pull/1044/commits/22ab7bbb0ef4bfe4a2539fe7522e6761733ba810 - decided to add an optional prop to handle both cases. Hope it makes sense.